spec/mailers/previews/bill_mailer_preview.rb

```diff
@@ -0,0 +1,16 @@
+# typed: strict
+# Preview all emails at http://localhost:3000/rails/mailers/bill_mailer
+class BillMailerPreview < ActionMailer::Preview
+  extend T::Sig
+
+  sig { returns(ActionMailer::MessageDelivery) }
+  def bill
+    BillMailer.bill(
+      to_name: 'Bill',
+      to_email: 'bill@cru.org',
+      from: 'Vonette',
+      for_message: 'Helping me learn to backflip!',
+      count: 7
+    )
```

Comment:
will this be replaced by user input?

Reply:
The preview file is a cool thing that rails has built in for previewing Mailers. You define a call to a mailer action in this class and rails generates this for you: http://localhost:3001/rails/mailers/bill_mailer/bill.html